library/n2_protocol.hpp

```diff
@@ -8,14 +8,35 @@
 
 namespace ioremap { namespace elliptics { namespace n2 {
 
-struct data_in_file {
-	int fd = -1;
-	off_t local_offset = 0;
-	size_t fsize = 0;
-	int on_exit = 0;
+class data_in_file {
+public:
+	struct info_t {
+		int fd = -1;
+		off_t local_offset = 0;
+		size_t fsize = 0;
+	};
+
+	data_in_file() = default;
+	data_in_file(data_in_file &&) = default;
+	data_in_file &operator = (data_in_file &&) = default;
```

Comment:
Please, write `operator` and `=` together, like there:
https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types